### PR TITLE
added a file with my personal html page using N++

### PR DESCRIPTION
**Pull request by olivia7n**

### CODE REVIEW

File: docs/noah-dev-profile/noah-dev-profile.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <br>
+            <h2>Quick Facts</h2>
+            <ul>
+                <li>GitHub Profile Link: <a href="https://github.com/olivia7n" target="_blank">GitHub</a></li>
```
Comment (barrycumbie) [inline docs/noah-dev-profile/noah-dev-profile.html:13]: plz put your github username in the text of the link too...we are not linking to "Github" in general, but to your user profile

File: docs/noah-dev-profile/noah-dev-profile.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Noah's Dev Profile</title>
+    </head>
+    <body style="text-align: left; background-color: rgb(260, 100, 200)">
```
Comment (barrycumbie) [inline docs/noah-dev-profile/noah-dev-profile.html:7]: whew, inline style. Prob the only time I'll let you get away with this.

File: docs/noah-dev-profile/noah-dev-profile.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
...PwB8O3XiHxP4bVpbLV9QeWLTNPeVG83/R22yXUi7tqs/wDdXczbVWvh7WvEWrePvFl/4g1S5uNV1q8unup7mVmllmlZtzO7f71ccZTqT53ojvwNbk0oo++/+CcnwV+H3wK+FHie2/aAu/B7f8JZa2upLoq+Ov7Pn8lYllSK7t4st5i+arqn95v4mVdvwh44uPDGqfEjxDfeFdLuo9GmvJv7Nt5GeVobfLeUrOyruZV2/NWb8TPHmufFbxfqnibXLiO+1vVHaW4uCi/N8u3aqqqqq/LtVVX5a/VX4XftCfs5/HH9g7wf8JdP1+z+GejrpunWviDUtTgij1K6u4DG9w9vFEjv5jzK26V3Xd/dZfmWalOVP397nRhsVWdZu17/AIHwv8Mf+CXPxw+NXg/SvFGmeDNSh0PVtzwPcL5ErRb0RXVH2/fkfan8TbX+Xate/fBf/gjTpWi+EdQ174i+MLOSx01LprxLK4SCC1liZlaDzm/1su5GVlXavzIu75m2/VXjH9snRv20f2fviz4P8H6hqE+sfDnS/tWgeIbK/XT4tTiijVFdbeV0fbsR/wCBvnlX7rbWb5N/b5tfi5qX7DngeOTwr4q8D/D+CK1gvF1i+itZNbvVibLxWu5ZXi4L7pFauf8AeyfJL3TWtQvf2qtM+Lde0Pwnr3xW1yTwZa6pZ+F45f8AQLa/uBc3kyKiqrSsvy7nZdzKv97bXsn7C3w/8O+Pv2oPD0fjhvD83hH7FNLPFreuto9rcMsT+VF9oWJ2RvM2t93a23b/ABVkfsFfGD4V/A74la1qvxU+HOrfEPQb7T/IsLKw1RrFreZW+++1l3rt3fxfL/tVs/tSftZfBP4sfD6H/hAPg7qngPxLHdsrXCeJXvtPa2+bb+6fc3m/Mqs25V+Xd/F8vpRjJR5Evmc2F+qKXLUvfuec6lp/h7xN+0p4mubaFfB/hnUtSaLyrS4bUFs7dX+Xa/yedtVd33V3V9lfEvwN4b0/9ma3+Gvwz8A+IJr74lazap4W1PXdOEGueSqLPLO+yLYsTLsdIt27y5d/91a+Pfh74z1v4UyWPiDSrpdN1SHdLBOkS/ud67Wzu3K25fvLt/2a/QD9lb/grRp/iWbRNY8ffEq48P8AiK4tEhv4I/CEUei2flbkZt8TSytJLAqLuiRF/vbtqrWFaM+iPosdUWEpU/Zfmf/Z"
```
Comment (barrycumbie) [inline docs/noah-dev-profile/noah-dev-profile.html:9]: got to finish this out...need that base64 encoding of an image to drop in here. Will be super long.

File: docs/noah-dev-profile/noah-dev-profile.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                <li>Visual Studio Code Editor and Notepad ++</li>
+                <li>Google Chrome Browser</li>
+            </ul>
+            <h2>5 Things from Class</h2>
+            <ol>
+                <li>Base64</li>
+                <li>Pull Requests</li>
+                <li>Gist</li>
+                <li>Dev Ops</li>
+                <li>Bill Gates Dancing</li>
```
Comment (barrycumbie) [inline docs/noah-dev-profile/noah-dev-profile.html:32]: can you give us some links to these tools, resources?

**Review by barrycumbie — CHANGES_REQUESTED** [review]

**Comment by barrycumbie** [comment]
@olivia7n not bad, very close. 

Remember to be nice to ol @barrycumbie and provide in the comments here a link to your active test page (your github pages) and a link to the Nu checker for that test page.

**Comment by olivia7n** [comment]
https://validator.w3.org/nu/?doc=https%3A%2F%2Folivia7n.github.io%2Fdev-challenge-01%2F

Link to nu checker with live address

**Comment by olivia7n** [comment]
https://olivia7n.github.io/dev-challenge-01/
this is the link github pages gave me, but it doesn't look right

**Comment by olivia7n** [comment]
https://validator.w3.org/nu/?doc=https%3A%2F%2Folivia7n.github.io%2Fdev-challenge-01%2Fdocs%2Fnoah-dev-profile%2Fnoah-dev-profile.html

here is the updated nu link

**Comment by olivia7n** [comment]
https://olivia7n.github.io/dev-challenge-01/docs/noah-dev-profile/noah-dev-profile.html

This link works!

**Comment by barrycumbie** [comment]
@olivia7n so super close...

1. Here's the deal with the proper link to what I call your ```active test page```:  
https://olivia7n.github.io/dev-challenge-01/docs/noah-dev-profile/noah-dev-profile.html
> * notice that it points to your docs/ folder in the URL 'cuz on your gh fork, you chose to deploy from the ```main``` branch vs the ```docs```. No big deal but you do need to notice it. 
> * also, notice that you have to point to the full url (name your .html file) 'cuz the server looks for a page called ```index.html``` 

so, you just need to drop in a link on your dev profile to the nu checker. Here it is pointing to your active test page: 

https://validator.w3.org/nu/?doc=https%3A%2F%2Folivia7n.github.io%2Fdev-challenge-01%2Fdocs%2Fnoah-dev-profile%2Fnoah-dev-profile.html

⚠️ make sure and resolve those errors and warnings (think it is just missing a closing ```>``` on the img 

then, anticipate what that link will be once your page does get published 
> hint: it won't be ```olivia7n``` but it will be ```barrycumbie```
> hint hint: does ```barrycumbie.github.io``` have a ```docs``` in its pathway? 🙅🏼‍♂️

**Comment by olivia7n** [comment]
Added Nu Checker link and fixed issues with the image tag

**Comment by olivia7n** [comment]
https://olivia7n.github.io/dev-challenge-01/docs/noah-dev-profile/noah-dev-profile.html

**Comment by barrycumbie** [comment]
@olivia7n Did I say you were close before? Now you are so so so super very close!

One thing missing on your line 33...this link should point to THE FUTURE. This should point to your dev profile ONCE I MERGE it to my REPO. That will essentially move it from your ```olivia7n.github.io``` to my ```barrycumbie.github.io```

Look at Harris' line 46: https://barrycumbie.github.io/dev-challenge-01/dev-profile/harris-dev-profile.html

**Review by barrycumbie — APPROVED** [review]

**Comment by olivia7n** [comment]
Changed last little issue with Nu link

**Comment by barrycumbie** [comment]
I'm gonna approve this even though: 

a) this link don't work:  https://olivia7n.github.io/dev-challenge-01/docs/noah-dev-profile/noah-dev-profile.html 
> hint: not to docs/
b) your Nu link on your page won't work, I bet
> hint: I don't have the docs/ in my URL 

this will be something you gotta fix.

**Pull request merged**